<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMCsoftware</class>
 <widget class="QMainWindow" name="IMCsoftware">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>418</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Visby CF Bold</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>IMCSoftware</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagenes.qrc">
    <normaloff>:/ICONOS/Desktop/IMG-20170414-WA0009.jpg</normaloff>:/ICONOS/Desktop/IMG-20170414-WA0009.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>311</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="parametersBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>167</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Visby CF Bold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Clientes con cuota vencida el dia de la fecha</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QTableWidget" name="tableWidget_VencenHoy"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="parametersBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>581</width>
      <height>167</height>
     </rect>
    </property>
    <property name="title">
     <string>Cliente con cuota vencida</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tableWidget_Vencidos"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="imagenes.qrc">:/ICONOS/Desktop/IMC_Logo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuClientes">
    <property name="title">
     <string>Clientes</string>
    </property>
    <addaction name="actionNuevo_Cliente"/>
    <addaction name="separator"/>
    <addaction name="actionBuscar_Cliente"/>
    <addaction name="separator"/>
    <addaction name="actionVer_Lista_de_Clientes"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSensor_de_Huella">
    <property name="title">
     <string>Sensor de Huella</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="separator"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionConfigure"/>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuClientes"/>
   <addaction name="menuSensor_de_Huella"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNuevo_Cliente"/>
   <addaction name="actionBuscar_Cliente"/>
   <addaction name="actionVer_Lista_de_Clientes"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNuevo_Cliente">
   <property name="icon">
    <iconset resource="imagenes.qrc">
     <normaloff>:/ICONOS/Desktop/181501-interface/181501-interface/png/user-1.png</normaloff>:/ICONOS/Desktop/181501-interface/181501-interface/png/user-1.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo Cliente</string>
   </property>
   <property name="font">
    <font>
     <family>Visby CF Bold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionBuscar_Cliente">
   <property name="icon">
    <iconset resource="imagenes.qrc">
     <normaloff>:/ICONOS/Desktop/181501-interface/181501-interface/png/zoom-in.png</normaloff>:/ICONOS/Desktop/181501-interface/181501-interface/png/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Buscar Cliente</string>
   </property>
   <property name="font">
    <font>
     <family>Visby CF Bold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionVer_Lista_de_Clientes">
   <property name="icon">
    <iconset resource="imagenes.qrc">
     <normaloff>:/ICONOS/Desktop/181501-interface/181501-interface/png/document-3.png</normaloff>:/ICONOS/Desktop/181501-interface/181501-interface/png/document-3.png</iconset>
   </property>
   <property name="text">
    <string>Ver Lista de Clientes</string>
   </property>
   <property name="font">
    <font>
     <family>Visby CF Bold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About program</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset>
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;onnect</string>
   </property>
   <property name="toolTip">
    <string>Connect to serial port</string>
   </property>
   <property name="font">
    <font>
     <family>Visby CF Bold</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset>
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset>
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure</string>
   </property>
   <property name="toolTip">
    <string>Configure serial port</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset>
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;lear</string>
   </property>
   <property name="toolTip">
    <string>Clear data</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
